src/community/jms-cluster/activemqBroker/src/test/java/org/geoserver/web/Start.java

```diff
@@ -33,11 +35,15 @@
     private static final Logger log = LoggerFactory.getLogger(Start.class);
 
     public static void main(String[] args) {
+        // don't even think of serving more than XX requests in parallel...
+        // we have a limit in our processing and memory capacities
+        ThreadPoolExecutor tp = (ThreadPoolExecutor) Executors.newCachedThreadPool();
+        tp.setMaximumPoolSize(50);
```

Comment:
It wasn't before. I would say there are probably lots of ways to make the script more customizable. For now I think it best to focus on the upgrade itself and tackle such things as separate issues, otherwise this one could snowball.